src/bptl/tasks/forms.py

```diff
@@ -71,6 +70,99 @@ def __init__(self, *args, **kwargs):
         self.fields["alias"].required = False
 
 
+class BaseDefaultServiceFormset(BaseInlineFormSet):
+    def clean(self):
+        super().clean()
+
+        # selected callback - this is not validated because we don't have access to the
```

Comment:
Right - we have a form and a formset, the form is used for the `TaskMapping` model, the formset has the `DefaultService` inline-formset.

From the `DefaultService` formset we can't get to the (validated) `TaskMapping` form instance, they are handled in the view only, but they don't have any references to each other. So, we can't access `form.cleaned_data["callback"]` because we can't get to `form`.

Reply:
Thanks for the explanation